applications/MultiScaleApplication/custom_strategies/strategies/arc_length_diss_nrg_strategy.h

```diff
@@ -779,9 +779,7 @@ class ArcLengthDissNrgStrategy
 
         mpBuilderAndSolver->ResizeAndInitializeVectors(mpScheme, 
 					mpA, mpDx, mpb, 
-					model.Elements(), 
-					model.Conditions(), 
-					model.GetProcessInfo());
+					model);

```

Comment:
Just to clean the code, would you please change the `model` to `model_part`?

Reply:
Done !